frontend/www/js/omegaup/components/badge/Badge.test.ts

```diff
@@ -0,0 +1,17 @@
+import { shallowMount } from '@vue/test-utils';
+import expect from 'expect';
+import badge_Badge from './Badge.vue';
+
+describe('Badge.vue', () => {
+  it('Should display badge name', () => {
+    const badge_alias = 'contestManager';
```

Comment:
nit:

```suggestion
    const badgeAlias = 'contestManager';
```

las variables deberían ir en camelCase